### PR TITLE
hotfix - properly evaluating test parameter of configuration since las update including default configs.

### PR DESCRIPTION
**Pull request by nickjoanis**
Following a recent deployment, I have found that the default configuration introduced in the last merge makes the evaluation of test property of the config object always true (which is default).

It seems that updates were done to allow configurations to be passed in the moneryze method directly instead of using the init function.

Unfortunately, even if configurations are passed as a parameter, the test property is still evaluated on config.test (default object) which is never overwritten.

Therefore, I have added a statement to evaluate if configuration.test is set and then it updates the value of config.test. Not sure it's the most elegant way, but considering that the value is a boolean I wanted to make a strict check.

A recent release candidate where moneryze.init were removed (on moneryze version 1.0.9) defaulted to Moneris' test environment if production configuration was passed.

**TLDR: config.test by default is set to true and is never overwritten in production environment if you don't use the moneryze.init method.**

### CODE REVIEW

**Comment by nickjoanis** [comment]
I think that if we are looking at allowing moneryze.init to be skipped, we should implement a way to overwrite the default configurations completely with the passed object.

Then we will be able to standardize checks on config instead of having mixed checks on _config_ and _configuration_ objects.
